src/errors.js

```diff
@@ -0,0 +1,16 @@
+var debug = require('debug')('error');
+var errors = exports;
+
+
+errors.internal = function(err, req, res, next) {
+  debug(err.stack);
+
+  res
+    .status(500)
+    .send({
+      errors: [{
+        type: 'internal_error',
+        message: 'Internal Error'
+      }]
```

Comment:
The error response format probably needs some explanation: Errors responses could have multiple errors (for example, an invalid username and a missing password field in some request), thus the need for an array of errors. A `success: false` field seems redundant, since we know it is an error because of the status code. A `type` field would be for programmer use and `message` would be for display use.